packages/flutter_tools/templates/create/android.tmpl/.gitignore

```diff
@@ -1,5 +1,6 @@
 *.iml
 *.class
+**/bin/
```

Comment:
If I have a Java package named `com.example.bitcoin.bin.coolminingalgo`, this could be problematic. Don't we know where `bin` folders are created by the Android Studio build process, so we can make this more restrictive?